src/cluster/slot_import.cc

```diff
@@ -27,9 +27,8 @@ SlotImport::SlotImport(Server *svr)
       import_status_(kImportNone),
       import_fd_(-1) {
   std::lock_guard<std::mutex> guard(mutex_);
-  // Let db_ and metadata_cf_handle_ be nullptr, then get them in real time while use them.
+  // Let metadata_cf_handle_ be nullptr, then get them in real time while use them.
   // See comments in SlotMigrate::SlotMigrate for detailed reason.
-  db_ = nullptr;
```

Comment:
db_ always ref system db here? Would this cause bug in transaction?

Reply:
No, it won't. It's right for the migration should always use the raw DB instead of the wrapped one. Because we don't expect to read uncommitted writes when migrating.